File: backend/src/main/java/pl/maciej/matyjasek/voting/app/candidate/dto/CandidateDto.java
package pl.maciej.matyjasek.voting.app.candidate.dto;

import lombok.NonNull;
import lombok.Value;

import java.util.UUID;

@Value
public class CandidateDto {

	UUID uuid;
	@NonNull
	String firstName;
	@NonNull
	String lastName;
	Integer votesQuantity;
}

File: frontend/voting/src/app/vote/candidate/candidateDto.ts
export class CandidateDto {
  uuid: string;
  firstName: string;
  lastName: string;
  votesQuantity: number;


  constructor(uuid: string, firstName: string, lastName: string, votesQuantity: number) {
    this.uuid = uuid;
    this.firstName = firstName;
    this.lastName = lastName;
    this.votesQuantity = votesQuantity;
  }
}
